packages/snaps-controllers/src/test-utils/controller.ts

```diff
@@ -535,8 +543,10 @@ export const DEFAULT_ENCRYPTION_KEY_DERIVATION_OPTIONS = {
   },
 };
 
-const getSnapControllerEncryptor = () => {
+export const getSnapControllerEncryptor = () => {
   return {
+    encrypt,
```

Comment:
If we are going to add these they should have the key derivation parameters set in the same way they are forced set in the clients. See below

Reply:
Removed them again. I ran into a type error, but ended up solving it in another way (since this didn't solve it either).